modules/assim.sequential/R/sda.enkf_refactored.R

```diff
@@ -304,19 +310,26 @@ sda.enkf <- function(settings,
         restart.arg = NULL
       }
       
+       if(t == 1){
+       config.settings = settings 
+       config.settings$run$end.date = format(lubridate::ymd_hms(obs.times[t], truncated = 3), "%Y/%m/%d")} 
+       if(t != 1){config.settings = settings}
```

Comment:
As above. IMHO, this makes the binary logic here ("if t==1, do X, otherwise do Y") a bit clearer.

```suggestion
       if (t == 1) {
         config.settings = settings 
         config.settings$run$end.date = format(lubridate::ymd_hms(obs.times[t], truncated = 3), "%Y/%m/%d")
       } else {
         config.settings = settings
       }
```